src/pages/projectSettings/Tabs.tsx

```diff
@@ -127,10 +127,19 @@ export const ProjectSettingsTabs: React.FC<Props> = ({
           />
         )}
       />
-      <TabRoute
-        Component={NotificationsTab}
+      <Route
         path={routes.projectSettingsNotifications}
-        tab={ProjectSettingsTabRoutes.Notifications}
+        render={(props) => (
+          <NotificationsTab
+            {...props}
+            id={projectId || repoData?.projectRef?.id}
```

Comment:
This field isn't being used, just wanted to flag if it's necessary (all good if you're going to be using it for the subscriptions section)

Reply:
I think I might need it for subscriptions. I'll keep it for now but keep it in mind.